core/css/inputs.scss

```
@@ -323,6 +323,10 @@ input[type='reset'] {
 		background-color: var(--color-error) !important;
 		border-color: var(--color-error) !important;
 		color: #fff !important;
+		&:hover{
+			background-color: var(--color-error-hover) !important;
+			border-color: var(--color-primary-element) !important;
```

```suggestion
			border-color: var(--color-main-text) !important;
```

I think this is more consistent with our current other components